test/input_gen/transLayer_v2.py

```diff
@@ -64,6 +64,37 @@ def transpose_(weight):
     new_params = [transpose_(params[0]), transpose_(params[1]), bias]
     yield from new_params
 
+@register_for_((torch.nn.GRUCell))
+def gru_translate(model):
+    params = [(name, tensor.detach()) for name, tensor in model.named_parameters()]
+    bias = ("bias", params[2][1] + params[3][1])
+
+    # hidden, input -> input, hidden
+    def transpose_(weight):
+        return (weight[0], weight[1].transpose(1, 0))
+
+    # resetgate, inputgate, newgate -> inputgate, resetgate, newgate
+    def reorder_weight(weight_):
+        if (weight_[1].dim() == 2):
+            hidden_size = int(weight_[1].shape[1] / 3)
+        else:
+            hidden_size = int(weight_[1].shape[0] / 3)
+
+        import copy
```

Comment:
If there is a better way let me know

Reply:
you would want to use np.hsplit + np.hstack  (torch should have the same method defined as well)

Reply:
Thanks 👍